.github/workflows/image-kairos-ubuntu.yml

```diff
@@ -51,7 +48,7 @@ jobs:
         uses: docker/build-push-action@471d1dc4e07e5cdedd4c2171150001c434f0b7a4 # v6.15.0
         with:
           build-args: |
-            VERSION=${{ steps.version_tag.outputs.tag }}
+            VERSION=${{ steps.version_tag.outputs.tag }}-${{ github.repository_owner }}-kairos-ubuntu
```

Comment:
I guess it's Kairos deciding that `VERSION` here writes to `KAIROS_VERSION` so this is ok but I would have expected that to be something like "3.5.x" not whatever version scheme we decide